src/tests_exhaustive.c

```diff
@@ -186,17 +186,13 @@ typedef struct {
     secp256k1_ge pt[2];
 } ecmult_multi_data;
```

Comment:
We can probably remove this. In the old code, I only see this acting as `cbdata` (initialized using local scalars and points arrays) to the `ecmult_multi_var`